### DISCUSSION TITLE
Welcome to Sonny-1-2-in-Sonny-2017-Rework Discussions!

### DISCUSSION THREAD

**Discussion by ArdaUzun-55** [LakshmisKshatriya/Sonny-1-2-in-Sonny-2017-Rework#1] [Announcements]
### Hi all,
I want people to use this Discussion Forum in order to give me Feedback/Suggestions or just want to talk about my mod!

**Comment by MaheeGamage** [comment]
Thank you for creating this MOD since I really like the Sonny series. I'm playing sonny (2017) steam edition and I have added your MOD files. After I started a new game. I see my skill tree is somewhat packed and bottom skills are cut off. Is this normal or a bug? If it is the default view I suggest you to organize it with adding some spacing between all the skills. Maybe a scroll bar?
Again thank you for the MOD.
![image](https://github.com/LakshmisKshatriya/Sonny-1-2-in-Sonny-2017-Rework/assets/26676091/8eff9b0f-cd0c-41c1-bad4-5de3f5373f71)

**Reply by ArdaUzun-55** [reply]
Oh sorry I havent seen your message.
Yes its the default look for the first tree you have but only for this tree. Its looking like that because I dont really know how to make the underhalf of the abilities visible. But maybe I will change it in the future. It doesnt  have an effect on the gameplay which is why it was not my priority.
Thank you for playing, I hope you enjoy it.
